docs/ref/databases.txt

```diff
@@ -625,12 +625,14 @@ both MySQL and Django will attempt to convert the values from UTC to local time.
 Row locking with ``QuerySet.select_for_update()``
 -------------------------------------------------
 
-MySQL does not support the ``NOWAIT``, ``SKIP LOCKED``, and ``OF`` options to
-the ``SELECT ... FOR UPDATE`` statement. If ``select_for_update()`` is used
-with ``nowait=True``, ``skip_locked=True``, or ``of`` then a
-:exc:`~django.db.NotSupportedError` is raised.
-
-When using ``select_for_update()`` on MySQL, make sure you filter a queryset
+For MySQL versions <8.0.1 and MariaDB versions <10.3.0, the ``NOWAIT``,
+``SKIP LOCKED``, and ``OF`` options to the ``SELECT ... FOR UPDATE`` statement
+are not supported. If ``select_for_update()`` is used with ``nowait=True``,
+``skip_locked=True``, or ``of`` then a :exc:`~django.db.NotSupportedError` is raised.
+However, MySQL supports only ``NOWAIT`` and ``SKIP LOCKED`` options in
+versions >=8.0.1 and MariaDB supports only the ``NOWAIT`` option in versions >=10.3.0.
```

Comment:
It's quite complicated, IMO table will be clearer:
```
MySQL and MariaDB do not support some options to the ``SELECT ... FOR UPDATE``
statement. If ``select_for_update()`` is used with a not supported option then
a :exc:`~django.db.NotSupportedError` is raised.

=============== ========== ==========
Option          MariaDB    MySQL     
=============== ========== ==========
``SKIP LOCKED``            X (≥8.0.1)
``NOWAIT``      X (≥10.3+) X (≥8.0.1)
``OF``
=============== ========== ==========
```

Reply:
I agree with you. I suggest we use `10.3+` and `8.0.1+` to be consistent with your suggestion in the `querysets.txt` file.

Reply:
I proposed `≥` to be consistent with other tables :smile: , e.g. [database-functions](https://docs.djangoproject.com/en/2.2/ref/contrib/gis/db-api/#database-functions).

Reply:
Ahh... Okay, thanks.

Reply:
@kola-er I will push minor fixes in 2 minutes.

Reply:
@felixxm I pushed the fix already.